.github/workflows/ci.yml

```diff
@@ -85,3 +85,41 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  dokken-recipes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - alinux2
+          - centos7
+          #- centos8
+          - ubuntu18
+          - ubuntu20
+        suite: [ 'base' ]
+      fail-fast: false
+    steps:
+      - name: Check out code
+        uses: actions/checkout@main
+      - name: Install Chef
+        uses: actionshub/chef-install@main
+      - name: Test-Kitchen Converge
+        uses: actionshub/test-kitchen@main
+        with:
+          suite: ${{ matrix.suite }}
+          os: ${{ matrix.os }}
+          action: converge
+        env:
+          CHEF_LICENSE: accept-no-persist
+          KITCHEN_YAML: kitchen.docker.yml
+          KITCHEN_LOCAL_YAML: kitchen.recipes.yml
+        continue-on-error: false
+      - name: Test-Kitchen Verify
+        uses: actionshub/test-kitchen@main
+        with:
+          suite: ${{ matrix.suite }}
+          os: ${{ matrix.os }}
+          action: verify
```

Comment:
Why separate? By the way, this way you keep active docker containers after test execution.

Reply:
unified in the https://github.com/aws/aws-parallelcluster-cookbook/pull/1648